src/screen/mod.rs

```diff
@@ -8,14 +10,14 @@ use crate::core::booting::Booting;
 
 pub(super) fn plugin(app: &mut App) {
     app.init_state::<Screen>()
-        .enable_state_scoped_entities::<Screen>();
-
-    app.add_plugins((title::plugin, credits::plugin, playing::plugin));
+        .enable_state_scoped_entities::<Screen>()
+        .add_plugins((title::plugin, credits::plugin, playing::plugin));
```

Comment:
I like splitting the `app.` chains based on logical grouping. The first two methods are for setting up `Screen`, and the third method is for adding sub-plugins. This plugin is small enough that putting it all in one chain is still readable, but for larger plugins it may hurt readability.

Willing to compromise on this.

Reply:
Agreed on grouping in bigger plugins, although I would just place a comment above a logical group and still chain it.
But in this case, it should be all clear enough that the terse version makes it more lightweight.

Also willing to compromise, not a strong opinion.

Reply:
😈

Reply:
[`8666105` (#59)](https://github.com/TheBevyFlock/bevy-template/pull/59/commits/86661056d0ef97f89c520aa75b9e60ef5ce9063f)

Reply:
I'm convinced by the mini argument. Never using `app.` chains anywhere is superior.